src/core/public/ui_settings/ui_settings_client.ts

```diff
@@ -41,7 +41,7 @@ export class UiSettingsClient {
   private readonly defaults: UiSettingsState;
   private cache: UiSettingsState;
 
-  constructor(readonly params: UiSettingsClientParams) {
+  constructor(params: UiSettingsClientParams) {
```

Comment:
Couldn't find any usage of `UiSettingsClient.params` it seems like it wasn't meant to be a member property.